sql/derby/DerbyLexer.g4

```diff
@@ -0,0 +1,435 @@
+TRANSLATE:                                             'TRANSLATE';
+TRANSLATION:                                           'TRANSLATION';
+TRIGGER:                                               'TRIGGER';
+TRIM:                                                  'TRIM';
+TRUE:                                                  'TRUE';
+TRUNCATE:                                              'TRUNCATE';
+TYPE:                                                  'TYPE';
+UCASE:                                                 'UCASE';
+UNCOMMITTED:                                           'UNCOMMITTED';
+UNION:                                                 'UNION';
+UNIQUE:                                                'UNIQUE';
+UNKNOWN:                                               'UNKNOWN';
+UPDATE:                                                'UPDATE';
+UPPER:                                                 'UPPER';
+UR:                                                    'UR';
+USAGE:                                                 'USAGE';
+USER:                                                  'USER';
+USING:                                                 'USING';
+VALUES:                                                'VALUES';
+VAR_POP:                                               'VAR_POP';
+VAR_SAMP:                                              'VAR_SAMP';
+VARCHAR:                                               'VARCHAR';
+VARYING:                                               'VARYING';
+VIEW:                                                  'VIEW';
+WHEN:                                                  'WHEN';
+WHENEVER:                                              'WHENEVER';
+WHERE:                                                 'WHERE';
+WINDOW:                                                'WINDOW';
+WITH:                                                  'WITH';
+WORK:                                                  'WORK';
+WRITE:                                                 'WRITE';
+XML:                                                   'XML';
+XMLEXISTS:                                             'XMLEXISTS';
+XMLPARSE:                                              'XMLPARSE';
+XMLQUERY:                                              'XMLQUERY';
+XMLSERIALIZE:                                          'XMLSERIALIZE';
+YEAR:                                                  'YEAR';
+
+
+DUMMY: 'DUMMY';
+
+
+WHITE_SPACE:             [ \t\r\n]+                    -> skip;
+
+SQL_COMMENT:            '/*' (SQL_COMMENT | .)*? '*/' -> channel(HIDDEN);
+LINE_COMMENT:           '--' ~[\r\n]*                 -> channel(HIDDEN);
+
+// TODO: ID can be not only Latin.
+DOUBLE_QUOTE_ID:        '"' ~'"'+ '"';
+SINGLE_QUOTE:           '\'';
+
+ID:                     [A-Z_] ( [A-Z0-9_] )*;
+
+
+STRING_LITERAL       options { caseInsensitive=false; }     : '\'' (~'\'' | '\'\'')* '\'';
```

Comment:
Using this option is not incorrect but it looks redundant since there is no using of case sensitive literals in token definition.